src/main/resources/assets/computercraft/lua/rom/programs/rename.lua

```diff
@@ -6,4 +6,9 @@ end
 
 local sSource = shell.resolve( tArgs[1] )
 local sDest = shell.resolve( tArgs[2] )
+
+if fs.exists( sDest ) == true then
```

Comment:
This is the only check in this PR that's actually meaningful...
Please remove the "== true" though.